app/scripts/metamask-controller.js

```diff
@@ -744,10 +744,12 @@ export default class MetamaskController extends EventEmitter {
 
     // Accounts
     const hdKeyring = this.keyringController.getKeyringsByType('HD Key Tree')[0]
+    const simpleKeyPairKeyring = this.keyringController.getKeyringsByType('Simple Key Pair')[0]
     const hdAccounts = await hdKeyring.getAccounts()
+    const simpleKeyPairAccounts = await simpleKeyPairKeyring.getAccounts()
     const accounts = {
       hd: hdAccounts.filter((item, pos) => (hdAccounts.indexOf(item) === pos)).map((address) => ethUtil.toChecksumAddress(address)),
-      simpleKeyPair: [],
+      simpleKeyPair: simpleKeyPairAccounts.filter((item, pos) => (simpleKeyPairAccounts.indexOf(item) === pos)).map((address) => ethUtil.toChecksumAddress(address)),
```

Comment:
Interesting - I assume the `.filter((item, pos) => (simpleKeyPairAccounts.indexOf(item) === pos))` is for removing duplicates. I didn't know it was possible to import duplicate accounts :thinking:

Reply:
In testing I was unable to import duplicate accounts. Did you encounter this scenario, or was this filter just a precaution?

Reply:
Oh nevermind - I see now that it was likely just taken from the line above this one. Fair enough.